tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/controllers/EndringsmeldingNavController.kt

```diff
@@ -1,18 +1,28 @@
 package no.nav.amt.tiltak.deltaker.controllers
 
+import no.nav.amt.tiltak.common.auth.AuthService
+import no.nav.amt.tiltak.common.auth.Issuer
+import no.nav.amt.tiltak.core.port.ArrangorAnsattTilgangService
+import no.nav.amt.tiltak.deltaker.dto.EndringsmeldingDto
 import no.nav.amt.tiltak.deltaker.service.EndringsmeldingService
-import org.apache.commons.lang3.NotImplementedException
+import no.nav.security.token.support.core.api.ProtectedWithClaims
 import org.springframework.web.bind.annotation.*
 import java.util.*
 
 @RestController
 @RequestMapping("/api/nav-ansatt/endringsmelding")
 class EndringsmeldingNavController(
-	private val service: EndringsmeldingService
+	private val service: EndringsmeldingService,
+	private val arrangorTilgangService: ArrangorAnsattTilgangService,
+	private val authService: AuthService
 ) {
 
 	@GetMapping
-	fun hentEndringsmeldinger(@RequestParam("gjennomforingId") gjennomforingId: UUID) {
-		throw NotImplementedException("Not implemented yet")
+	@ProtectedWithClaims(issuer = Issuer.AZURE_AD)
+	fun hentEndringsmeldinger(@RequestParam("gjennomforingId") gjennomforingId: UUID) : List<EndringsmeldingDto> {
+		val ident = authService.hentPersonligIdentTilInnloggetBruker()
+		arrangorTilgangService.verifiserTilgangTilGjennomforing(ident, gjennomforingId)
```

Comment:
Dette vil ikke fungere så bra for nav ansatte siden `arrangorTilgangService` fungerer kun for tiltaksarrangør ansatte. 

`authService.hentPersonligIdentTilInnloggetBruker()` skal også kun brukes med TokenX

Reply:
Nei, det er sant.. vet ikke hva jeg tenkte